docs/_docs/continuous-integration/buddyworks.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "BuddyWorks"
+---
+
+[BuddyWorks][0] is a [Docker][1]-based CI server that you can set up in 15-20 minutes to build, test, and deploy your Jekyll websites. It supports [GitHub][2], [Bitbucket][3], and [GitLab][4] repositories, and can be installed on-premises or used in cloud. The following guide will show you how to set up a free environment to build and test your Jekyll project.
+
+[0]: https://buddy.works
+[1]: https://www.docker.com/
+[2]: https://github.com
+[3]: https://bitbucket.org/
+[4]: https://gitlab.com
+
+## 1. Getting started
+
+1. Log in at https://buddy.works with your GitHub/Bitbucket account or email
+2. Choose your Git provider and select or push your Jekyll Project
+3. Create a new pipeline and set the trigger mode to 'On every push'
+4. Add and configure the Jekyll action and save the pipeline
+
+## 2. How it works
+
+Whenever you make a push to the selected branch, the Jekyll action runs `jekyll build` in an isolated [Jekyll Docker image][5]. The output is generated to the `/filesystem` directory, and can be further deployed to FTP/SFTP and IaaS services. You can add your own commands, install additional packages, attach services, and run Selenium tests, as well as add other actions down the pipeline, eg. a Slack notification or an SSH script that will restart your server.
+
+![Jekyll Build](https://buddy.works/data/blog/_images/buddyworks-jekyll-small.png)
+
+[5]: https://hub.docker.com/r/jekyll/jekyll/
+
+## 3. Using YAML for configuration
+
+If you prefer configuration as code over GUI, you can generate `buddy.yml` that will create a pipeline with the Jekyll action once you push it to the target branch:
```

Comment:
the only thing that still bugs me is that this should say `generate a buddy.yml`. otherwise, all good!